tests/unittests/Foundation/NSUndoManagerTests.mm

```diff
@@ -0,0 +1,239 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
```

Comment:
>Copyright [](start = 3, length = 9)

Please use new copyright header in the files you are touching.  This line should read "Copyright (c) Microsoft. All rights reserved."